lib/+mwbs/+Visualizers/+robotVisualizer/RobotVisualizer.m

```diff
@@ -1,92 +1,100 @@
 classdef RobotVisualizer < matlab.System
     % RobotVisualizer matlab.System handling the robot visualization.
     % go in app/robots/iCub*/initRobotVisualizer.m to change the setup config
-
+    
```

Comment:
Are these trailing whitespace intentional?

Reply:
Not at all. I used the Matlab editor indentation. :(
I could change it with another editor.